script/tool/lib/src/firebase_test_lab_command.dart

```diff
@@ -110,31 +109,34 @@ class FirebaseTestLabCommand extends PackageLoopingCommand {
       print('');
       if (exitCode == 0) {
         print('Firebase project configured.');
-        return;
```

Comment:
This is the actual fix for the bug, but the `Completer` was needlessly complex so I cleaned it up as well.